src/detail_screen/anime.js

```diff
@@ -0,0 +1,144 @@
+    video_url: 'https://www.youtube.com/watch?v=ocQ6PDiP014'
+  },
+
+  shownDialog: false
+}
+
+const reducer = (state, action) => {
+  if (action === 'ANIME/SHOW_DIALOG') {
+    return { ...state, shownDialog: true }
+
+  } else if (action === 'ANIME/HIDE_DIALOG') {
+    return { ...state, shownDialog: false }
+  }
+
+  return state
+}
+
+const Anime = props => {
+  const [state, dispatch] = React.useReducer(reducer, initialState)
+
+  return <>
+    <Header />
+
+    <main>
+      <section className='paper'>
+        <div id='title'>
+          <h1>{ state.anime.name }</h1>
+
+          <IconButton icon='star_rate' onClick={ () => console.log('CLICK') } />
+        </div>
+
+        <img id='main-image' src={ state.anime.image_url } alt='' />
+
+        <p>Episodes: { state.anime.episodes }</p>
+        <p>Author: { state.anime.author }</p>
+        <p>Genres: { state.anime.genres.join(', ') }</p>
+        <p>Published: { state.anime.published }</p>
+      </section>
+
+      <section>
+        <div className='paper'>
+          <h2>Rank #{ state.anime.rank }</h2>
+        </div>
+
+        <div className='paper' id='rating'>
+          <div>
+            <h2>Score</h2>
+            <p>{ state.anime.mean_score }<span>{ state.anime.total_votes } votes</span></p>
+          </div>
+
+          <div className='divider'></div>
+
+          <div>
+            <h2>Status</h2>
+            <p>{ state.anime.status }</p>
+          </div>
+
+          <div>
+            <h2>Your rate</h2>
+            <p>{ state.anime.personal_score }</p>
+          </div>
+
+          <div>
+            <IconButton icon='edit' onClick={ () => dispatch('ANIME/SHOW_DIALOG') } />
+          </div>
+        </div>
+
+        <div className='paper' id='description'>
+          <h2>Description</h2>
+          <p>{ state.anime.description }</p>
+        </div>
+
+        <iframe src='https://www.youtube.com/embed/ocQ6PDiP014' title='YouTube video' frameBorder='0' allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture' allowFullScreen></iframe>
+      </section>
+    </main>
+
+    <ChangeRateDialog open={ state.shownDialog } onClose={ () => dispatch('ANIME/HIDE_DIALOG') } />
+  </>
+}
+
+const Header = props => {
```

Comment:
Já que props não é utilizado podemos obstruí-lo
```suggestion
const Header = () => {
```